example/model.json

```diff
@@ -1,7 +1,46 @@
 {
-    "method": {
-        "...": "include(method.json)"
-    },
-    "subtype": "gga",
-    "type": "dft"
+    "name": "KS-DFT",
+    "slug": "ks_dft",
+    "modelGraph": [
+        {
+            "name": "Kohn-Sham Density Functional Theory",
+            "flowchartId": "u123fndff333",
+            "head": true,
+            "workflowUnitId": "badd104b2e31300c61eb194d"
+        }
+    ],
+    "units": [
+        {
+            "flowchartId": "u123fndff333",
+            "modelPath": "pb/qm/dft/ksdft/gga",
+            "subtype": {
+                "name": "Generalized Gradient Approximation",
+                "slug": "gga"
+            },
+            "functional": {
+                "name": "Perdew-Burke-Ernzerhof",
+                "slug": "pbe",
+                "components": [
```

Comment:
Is PBE adjustable in terms of the fractions below? If not, should we consider separating the components as "standard" data (lookup) and not including them here to avoid repetition?

Reply:
No, it's not adjustable, the factor only comes into play when we add exact exchange or explicit correlation. I found one example among the GGAs which uses a fraction (HLE16: 1.25 HCTH/407 exchange + 0.5 HCTH/407 correlation). Agreed, we should have a lookup for the static data (conventional XC functionals such as PBE, BLYP, etc).